src/sympc/tensor/static.py

```diff
@@ -117,6 +121,27 @@ def cat_share_tensor(session_uuid_str: str, *shares: Tuple[ShareTensor]) -> Shar
     result.tensor = torch.cat([share.tensor for share in shares])
     return result
 
+def cat_replicatedShare_tensor(session_uuid_str: str, *shares: Tuple[ReplicatedSharedTensor]) -> ReplicatedSharedTensor:
+    """Helper method that performs torch.cat on the replicated shares of the Tensors.
+
+    Args:
+        session_uuid_str (str): UUID to identify the session on each party side.
+        shares (Tuple[ShareTensor]): Replicated shares of the tensors to be concatenated.
+
+    Returns:
+        ReplicatedSharedTensor: Respective replicated shares after concatenation
+    """
+    
+    session = get_session(session_uuid_str)
+    result = ReplicatedSharedTensor(session_uuid=UUID(session_uuid_str), config=session.config)
+    
+    cat_result = [torch.tensor([]).type(torch.LongTensor) for _ in range(len(shares[0].shares))]
```

Comment:
MAYBE: Could you save the `len(shares[0].shares)` in a variable and then use it here and on line 140?

Reply:
Yes, nice catch, that will more efficient. 👍